telemetry/csharp/README.md

```diff
@@ -0,0 +1,22 @@
+# C# AWS Toolkit Telemetry Generator
+
+This generator produces code that generates and records telemetry events. It is intended for use in C# systems like the AWS Toolkit for Visual Studio.
+
+At this time, the generated code does not function in a standalone capacity. It requires datatypes (within a `Amazon.ToolkitTelemetry` namespace) that are produced when an SDK-style Telemetry Client is generated. This code lives within the Toolkit repo at this time. See Roadmap for future plans in this space.
+
+## Generator Usage
+
+See [Options file](ToolkitTelemetryGenerator/Options.cs) for details.
```

Comment:
Can you please put it here instead

Reply:
The intent is to avoid documentation that can get out of sync with the program.

Reply:
Right but I can't tell what that means because I'm not familiar with the library